### PR TITLE
Add turbo-seti and blimpy

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge/help-python, ready for review!

File: recipes/blimpy/LICENSE [hunk 1]
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```
Comment (carterbox) [inline recipes/blimpy/LICENSE:1]: The source distribution already contains this license file, so it is not necessary to contain license manually.

File: recipes/blimpy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    - stix --help
+    - peek --help
+    - srcname --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ucberkeleyseti/blimpy
+  summary: Python utilities for Breakthrough Listen SETI observations
+  license: MIT
```
Comment (carterbox) [inline recipes/blimpy/meta.yaml:78]: This license type is incorrect.

File: recipes/blimpy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    - stax = blimpy.stax:cmd_tool
+    - stix = blimpy.stix:cmd_tool
+    - peek = blimpy.peek:cmd_tool
+    - srcname = blimpy.srcname:cmd_tool
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```
Comment (carterbox) [inline recipes/blimpy/meta.yaml:33]: I don't think a compiler is needed. If not, then this should be noarch:python like on PyPI.

File: recipes/blimpy/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+  host:
+    - pip
+    - pytest-runner
+    - python
+  run:
+    - astropy
+    - cython
```
Comment (carterbox) [inline recipes/blimpy/meta.yaml:40]: Maybe open an issue upstream because it looks like cython is unused in the current version.

**Review by carterbox — CHANGES_REQUESTED** [review]

File: recipes/blimpy/LICENSE (hunk 1 above)
Comment (jan-janssen) [reply recipes/blimpy/LICENSE]: Done

File: recipes/blimpy/meta.yaml (hunk 2 above)
Comment (jan-janssen) [reply recipes/blimpy/meta.yaml]: Done

File: recipes/blimpy/meta.yaml (hunk 3 above)
Comment (jan-janssen) [reply recipes/blimpy/meta.yaml]: switched to noarch

File: recipes/blimpy/meta.yaml (hunk 4 above)
Comment (jan-janssen) [reply recipes/blimpy/meta.yaml]: I opened an issue upstream to address this https://github.com/UCBerkeleySETI/blimpy/pull/265

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes ready for review

**Comment by jan-janssen** [comment]
@carterbox I fixed the issues - can you take another look?

**Pull request merged**